src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -30,6 +30,7 @@
 use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariantInterface;
 use Sylius\Component\Core\Model\TaxonInterface;
+use Symfony\Component\HttpFoundation\Request as HttpRequest;
```

Comment:
I don't see a naming collision, therefore `as` is not necessary.
```suggestion
use Symfony\Component\HttpFoundation\Request;
```